kitsune/sumo/static/js/wiki_search.js

```diff
@@ -0,0 +1,78 @@
+(function($, _) {
```

Comment:
Should you be calling `k.nunjucksEnv` `_`? That is pretty confusing considering that is also Underscore and usual name.